jujugui/static/gui/src/app/components/account/payment/charges/charges.js

```diff
@@ -0,0 +1,262 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2017 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('account-payment-charges', function() {
+
+  juju.components.AccountPaymentCharges = React.createClass({
+    displayName: 'AccountPaymentCharges',
+
+    propTypes: {
+      acl: React.PropTypes.object.isRequired,
+      addNotification: React.PropTypes.func.isRequired,
+      getCharges: React.PropTypes.func.isRequired,
+      getReceipt: React.PropTypes.func.isRequired,
```

Comment:
In the account the payment sections are only displayed under a feature flag under which those methods don't exist, when the sections are displayed the methods do exist (same flag).